src/pages/CourseDetail/index.js

```diff
@@ -56,14 +59,36 @@ const coursesData = courseDetail[0].coursesSection;
 
 function CourseDetail(props) {
   const classes = useStyles();
-  const history = useHistory();
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   return (
     <>
       <div id="back-to-top-anchor"></div>
       <ScrollToTop />
       <div className={classes.navigation}>
-        <ArrowBackIcon className={classes.svg} onClick={() => history.goBack()} />
-        <MoreVertIcon className={classes.svg} />
```

Comment:
To me `history.goBack()` was  much better than always routing it to `/`.
 You can revert that code or if there was any problem with the previous code you can tell me that too.

Reply:
Will revert this. Yeah actually go back makes more sense as we have swipeable routes.